featuretools/tests/computational_backend/test_feature_set_calculator.py

```diff
@@ -1293,3 +1294,26 @@ def error(s):
     # Calculating without precalculated features should error.
     with pytest.raises(RuntimeError, match=error_msg):
         FeatureSetCalculator(pd_es, feature_set=FeatureSet([direct])).run(instance_ids)
+
+
+def test_nunique_nested_with_agg_bug(pd_es):
+    """Pandas 2.2.0 has a bug where pd.Series.nunique produces columns with
+    the category dtype instead of int64 dtype, causing an error when we attempt
+    another aggregation"""
```

Comment:
The error message showing up was `TypeError: category type does not support sum operations`, but this is actually a bug in NumUnique, which was returning data with dtype `category` with numeric categories. The error would eventually show up because the mean, sum, std, etc aggregation functions cannot be made on non numeric data.